### ISSUE TITLE
not working on neovim

### ISSUE THREAD

**Issue by mori97** [julia-vscode/LanguageServer.jl#318]
I followed the settings in wiki(https://github.com/JuliaEditorSupport/LanguageServer.jl/wiki/Vim-and-Neovim) but it seems not working and i got error message "[LC] Language server julia exited unexpectedly: Unable to read from language server".

The setting in wiki is
```
let g:LanguageClient_serverCommands = {
        \ 'julia': ['/Applications/Julia-1.0.app/Contents/Resources/julia/bin/julia', '--startup-file=no', '--history-file=no', '-e', '
        \       import LanguageServer;
        \       server = LanguageServer.LanguageServerInstance(stdin, stdout, false);
        \       server.runlinter = true;
        \       run(server);
        \   ']
        \}
```
but I found that LanguageServerInstance requires 6 arguments.
I got 
```
julia> server = LanguageServerInstance(stdin, stdout, false);
ERROR: MethodError: no method matching LanguageServerInstance(::Base.TTY, ::Base.TTY, ::Bool)
Closest candidates are:
  LanguageServerInstance(::Any, ::Any, ::Bool, ::Any, ::Any, ::Any) at /home/mori/.julia/dev/LanguageServer/src/languageserverinstance.jl:21
Stacktrace:
 [1] top-level scope at none:0
```
when i tried to execute the code with julia interpreter.

**Comment by davidanthoff** [comment]
This is a bug in the description for neovim, there are now some new arguments that one needs to pass.

I'll try to update the doc once we have a more stable version and have shipped v0.11.0 of the VS Code extension.

**Comment by oxinabox** [comment]
It should be a matter of calling this script (or something very like it)
https://github.com/JuliaEditorSupport/julia-vscode/blob/master/scripts/languageserver/main.jl

with the arguments it expects,
(which can be seen in said script)
